output/writer.go

```diff
@@ -1,8 +1,18 @@
 package output
 
+import "time"
+
 type Writer interface {
-	init(string) error
-	writeHeaders(string) error
-	writeData(string) error
+	Init(string) error
+	WriteEntry(TimedLocationEntry) error
+	writeEntryBatch([]TimedLocationEntry) error
 	finalize() error
 }
+
+type TimedLocationEntry struct {
+	timestamp time.Time
+	entityId  string
+	deviceId  string
```

Comment:
we don't need entityId and deviceId, probably in the future we can add feature to paste additional parameterized fields, but right now it is not the case

Reply:
Done